qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains functions to construct many-body observables whose expectation
+values can be used to simulate molecular properties.
+"""
+# pylint: disable=too-many-arguments, too-few-public-methods
+import os
+import numpy as np
+from openfermion.hamiltonians import MolecularData
+from openfermion.ops import FermionOperator
+from openfermion.transforms import bravyi_kitaev, jordan_wigner
+
+from . import structure
+
+
+def s2_me_table(sz, n_spin_orbs):
+    r"""Generates the table of the matrix elements
+    :math:`\langle \alpha, beta \vert \hat{s}_1 \cdot \hat{s}_2 \vert \gamma, \delta \rangle`
+    of the two-particle spin operator :math:`\hat{s}_1 \cdot \hat{s}_2`.
+
+    The matrix elements are evaluated using the expression,
+
+    .. math:
+
+        \langle \alpha, \beta \vert \hat{\bf{s}}_1 \cdot \hat {\bf{s}}_2
+        \vert \gamma, \delta \rangle = && \delta_{\alpha,\delta} \delta_{\beta,\gamma} \times \\
+        && \left( \frac{1}{2} \delta_{m_\alpha, m_\delta+1} \delta_{m_\beta, m_\gamma-1}
+        + \frac{1}{2} \delta_{m_\alpha, m_\delta-1} \delta_{m_\beta, m_\gamma+1}
+        + m_\alpha m_\beta \delta_{m_\alpha, m_\delta} \delta_{m_\beta, m_\gamma} \right),
```

Comment:
@agran2018, this is not showing up for me when I build the docs 🤔

Reply:
will check this

Reply:
Working now!.